docs/guides/lint-arazzo.md

```diff
@@ -0,0 +1,73 @@
+# Lint Arazzo with Redocly CLI
+
+Arazzo is a open standard for describing multi-step API calls in sequence.
```

Comment:
Maybe we could add the link to the Arazzo  spec?
https://spec.openapis.org/arazzo/latest.html#arazzo-specification

Reply:
Added by @JLekawa , thanks both of you!